WPFWijkagent/WPFWijkagent/Session/ASession.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace WijkagentWPF.Session
+{
+    public abstract class ASession
+    {
+        public const char Separator = '|';
+        public string Key { get; private set; }
+
+        public ASession(string key) => Key = key;
+
+        public abstract void Load(string input);
```

Comment:
Bij de implementaties van Load en Save missen comments. Ik weet het niet zeker maar misschien kan je bij de abstracte methode comments toevoegen en dat die dan geerfd worden door de implementaties?

Reply:
toegevoegd in ASession. er staan verder nergens directe references naar de implementaties dus ik denk niet dat comments daar nodig zijn